tests/simpleharness/simpleharness.csproj

```diff
@@ -6,7 +6,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
```

Comment:
I see.  I think rather than detecting version, you should detect whether or not the API is present and then show a warning.  That way, you don't have to worry about what version was compiled against or someone inserting privates, etc.